src/pso.rs

```diff
@@ -34,6 +34,13 @@ impl Shader {
         })
     }
 
+    pub fn from_raw(data: &[u8]) -> Self {
```

Comment:
should `Shader` have a generic lifetime `<'a>` inherited from a slice here?

Reply:
as the rest of the API does not track lifetimes I would keep it as is.